### PR TITLE
Typo fixed and this time I provide my cSpell.json

### PR DESCRIPTION
**Pull request by xulike666**
Signed-off-by: Aaron.L.Xu <likexu@harmonycloud.cn>

**- What I did**

I was very annoyed by those kind of typo mistakes and maybe this file (https://github.com/xulike666/vscode-spell-checker-helper/blob/master/docker/spellcheck.txt) could help our developers in a self-discipline and distributed way.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: cli/command/image/build.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func NewBuildCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 // lastProgressOutput is the same as progress.Output except
 // that it only output with the last update. It is used in
-// non terminal scenarios to depresss verbose messages
+// non terminal scenarios to depresses verbose messages
```
Comment (cpuguy83) [inline cli/command/image/build.go]: suppress?

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

File: cli/command/image/build.go (hunk 1 above)
Comment (vdemeester) [inline cli/command/image/build.go]: ah lol :joy:

File: man/dockerd.8.md [hunk 2]
```diff
@@ -404,7 +404,7 @@ Managing the thin-pool outside of Engine makes for the most feature-rich
 method of having Docker utilize device mapper thin provisioning as the
 backing storage for Docker containers. The highlights of the lvm-based
 thin-pool management feature include: automatic or interactive thin-pool
-resize support, dynamically changing thin-pool features, automatic thinp
+resize support, dynamically changing thin-pool features, automatic thin
```
Comment (cpuguy83) [inline man/dockerd.8.md]: `thinp` is correct

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**